services/common/src/components/forms/SteppedForm.tsx

```diff
@@ -1,54 +1,57 @@
-import { Button, Col, Form, Menu, Popconfirm, Row } from "antd";
+import React, { FC, ReactElement, useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { getFormSyncErrors, getFormValues, submit } from "redux-form";
+import { Button, Col, Menu, Popconfirm, Row, StepProps } from "antd";
```

Comment:
Basically, SteppedForm in this state:
- makes its own FormWrapper with the settings that are ideal for it
- checks for errors
- passes down isEditMode to FormWrapper to enable a view mode
- transforms the values according to a function prop